airflow/example_dags/example_kubernetes_executor_config.py

```diff
@@ -25,147 +25,133 @@
-        def task_with_template():
-            print_stuff()
-
-        task_with_template = task_with_template()
-        # [END task_with_template]
-
-        # [START task_with_sidecar]
-        executor_config_sidecar = {
-            "pod_override": k8s.V1Pod(
-                spec=k8s.V1PodSpec(
-                    containers=[
-                        k8s.V1Container(
-                            name="base",
-                            volume_mounts=[k8s.V1VolumeMount(mount_path="/shared/", name="shared-empty-dir")],
-                        ),
-                        k8s.V1Container(
-                            name="sidecar",
-                            image="ubuntu",
-                            args=["echo \"retrieved from mount\" > /shared/test.txt"],
-                            command=["bash", "-cx"],
-                            volume_mounts=[k8s.V1VolumeMount(mount_path="/shared/", name="shared-empty-dir")],
-                        ),
-                    ],
-                    volumes=[
-                        k8s.V1Volume(name="shared-empty-dir", empty_dir=k8s.V1EmptyDirVolumeSource()),
-                    ],
-                )
-            ),
-        }
-
-        @task(executor_config=executor_config_sidecar)
-        def test_sharedvolume_mount():
-            """
-            Tests whether the volume has been mounted.
-            """
-            for i in range(5):
-                try:
-                    return_code = os.system("cat /shared/test.txt")
-                    if return_code != 0:
-                        raise ValueError(f"Error when checking volume mount. Return code {return_code}")
-                except ValueError as e:
-                    if i > 4:
-                        raise e
-
-        sidecar_task = test_sharedvolume_mount()
-        # [END task_with_sidecar]
-
-        # Test that we can add labels to pods
-        executor_config_non_root = {
-            "pod_override": k8s.V1Pod(metadata=k8s.V1ObjectMeta(labels={"release": "stable"}))
-        }
-
-        @task(executor_config=executor_config_non_root)
-        def non_root_task():
-            print_stuff()
-
-        third_task = non_root_task()
-
-        executor_config_other_ns = {
-            "KubernetesExecutor": {"namespace": "test-namespace", "labels": {"release": "stable"}}
-        }
-
-        @task(executor_config=executor_config_other_ns)
-        def other_namespace_task():
-            print_stuff()
-
-        other_ns_task = other_namespace_task()
-
-        start_task >> volume_task >> third_task
-        start_task >> other_ns_task
-        start_task >> sidecar_task
-        start_task >> task_with_template
 except ImportError as e:
     log.warning("Could not import DAGs in example_kubernetes_executor_config.py: %s", str(e))
     log.warning("Install kubernetes dependencies with: pip install apache-airflow['cncf.kubernetes']")
+
+
+with DAG(
+    dag_id='example_kubernetes_executor_config',
```

Comment:
Should we just have a single example dag file for all Kubernetes Executor tasks? 

WDYT @ephraimbuddy @jedcunningham

Reply:
Yep, I think so: https://github.com/apache/airflow/pull/19355#pullrequestreview-797771124

Reply:
Cool, your "comment" went unnoticed in between review comments :D 

@ephraimbuddy Can you update please